src/dispatch/plugins/dispatch_aws/plugin.py

```diff
@@ -73,6 +73,10 @@ def consume(self, db_session: Session, project: Project) -> None:
                     log.exception(
```

Comment:
Should we change this to a warning then?

Reply:
True, changed that.